auth.c

```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2012 to the present time.
+*/
```

Comment:
This should probably be indented by a space. It's like the most minor thing ever, but it's irrationally annoying when the `*`s aren't in line 😜